stanford/data-ansible.cfg

```diff
@@ -5,6 +5,7 @@ host_key_checking = False
 hostfile=./ec2.py
 jinja2_extensions=jinja2.ext.do
 log_path=~/data-ansible.log
+roles_path = ../playbooks/roles:./roles
```

Comment:
what's this syntax mean path:path

Reply:
It's the same as the shell `$PATH` variable [1].

[1] https://unix.stackexchange.com/questions/26047/how-to-correctly-add-a-path-to-path

Reply:
aka: colon-delimited list

Reply:
Sadly, you can't use variables in an ansible config file.
Also, 3 of these files will be deleted with #110